std/math.d

```diff
@@ -4515,20 +4515,34 @@ public:
     version (IeeeFlagsSupport)
     {
 
-     /// The result cannot be represented exactly, so rounding occurred.
-     /// (example: x = sin(0.1); )
+     /**
+      * The result cannot be represented exactly, so rounding occurred.
+      * Example: `x = sin(0.1);`
+      */
```

Comment:
Yes and yes. You're free to do as you wish in your own code, but for Phobos it'll be the `/** */` style.